sw/host/opentitanlib/BUILD

```diff
@@ -13,6 +13,7 @@ rust_library(
         "src/app/command.rs",
         "src/app/conf.rs",
         "src/app/mod.rs",
+        "src/bootstrap/legacy.rs",
```

Comment:
I think this is out of place.

Reply:
It is out of place; it was necessary to get overall coverage to build.  It's been resolved after submitting #9338 and should is no longer part of this PR.